src/index.html

```diff
@@ -9,6 +9,10 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
+      <div class="stopwatch__center"></div>
+      <div class="stopwatch__seconds stopwatch__seconds--5"></div>
+      <div class="stopwatch__seconds stopwatch__seconds--60"></div>
+    </div>
```

Comment:
it will look better if the classes are shortened

```suggestion
    <div class="stopwatch">
      <div class="stopwatch__center"></div>
      <div class="stopwatch__seconds-hend"></div>
      <div class="stopwatch__hover-hand"></div>
    </div>
```